src/tests/test_jsproxy.py

```diff
@@ -38,6 +38,21 @@ def test_jsproxy_dir(selenium):
     assert set1.issuperset(jsproxy_items)
     assert set1.issuperset(callable_items)
     assert set1.isdisjoint(a_items)
+    selenium.run_js(
+        """
+        window.a = [0,1,2,3,4,5,6,7,8,9];
+        a[":"] = 0;
+        a["/"] = 0;
```

Comment:
Could you also add an example with a 2 char integer that is filtered and a multi character key that's not?

Maybe also a test with a non ascii key?